src/js/fc.js

```diff
@@ -893,4 +893,93 @@ const FC = {
     getSliderDefaults() {
         return this.DEFAULT_TUNING_SLIDERS;
     },
+
+    RATES_TYPE: {
+        BETAFLIGHT: 0,
+        RACEFLIGHT: 1,
+        KISS: 2,
+        ACTUAL: 3,
+        QUICKRATES: 4,
+    },
+
+    getCurrentRates() {
+
+        const currentRates = {
+            roll_rate:          FC.RC_TUNING.roll_rate,
+            pitch_rate:         FC.RC_TUNING.pitch_rate,
+            yaw_rate:           FC.RC_TUNING.yaw_rate,
+            rc_rate:            FC.RC_TUNING.RC_RATE,
+            rc_rate_yaw:        FC.RC_TUNING.rcYawRate,
+            rc_expo:            FC.RC_TUNING.RC_EXPO,
+            rc_yaw_expo:        FC.RC_TUNING.RC_YAW_EXPO,
+            rc_rate_pitch:      FC.RC_TUNING.rcPitchRate,
+            rc_pitch_expo:      FC.RC_TUNING.RC_PITCH_EXPO,
+            superexpo:          FC.FEATURE_CONFIG.features.isEnabled('SUPEREXPO_RATES'),
+            deadband:           FC.RC_DEADBAND_CONFIG.deadband,
+            yawDeadband:        FC.RC_DEADBAND_CONFIG.yaw_deadband,
+            roll_rate_limit:    FC.RC_TUNING.roll_rate_limit,
+            pitch_rate_limit:   FC.RC_TUNING.pitch_rate_limit,
+            yaw_rate_limit:     FC.RC_TUNING.yaw_rate_limit,
+        };
+
+        if (semver.lt(FC.CONFIG.apiVersion, "1.7.0")) {
+            currentRates.roll_rate = FC.RC_TUNING.roll_pitch_rate;
+            currentRates.pitch_rate = FC.RC_TUNING.roll_pitch_rate;
+        }
+
+        if (semver.lt(FC.CONFIG.apiVersion, "1.16.0")) {
+            currentRates.rc_rate_yaw = currentRates.rc_rate;
+        }
+
+        if (semver.gte(FC.CONFIG.apiVersion, "1.20.0")) {
+            currentRates.superexpo = true;
+        }
+
+        if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_36)) {
+            $('.pid_tuning input[name="sensitivity"]').hide();
+            $('.pid_tuning .levelSensitivityHeader').empty();
```

Comment:
Adding UI with one tab in the `fc.js` ? This is not the way to go.

Reply:
Good find, moved this part back to pid_tuning tab.